src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -489,8 +489,8 @@ def _first_half_condition(self):
         return Function(
             "less",
             [
-                Function("toDateTime", [self._get_middle()]),
                 self.builder.column("timestamp"),
+                Function("toDateTime", [self._get_middle()]),
```

Comment:
oh probs needs to be fixed in datasets/metrics.py too

Reply:
:+1: I'll fix it in this PR